config/initializers/lograge.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+Rails.application.configure do
+  config.lograge.enabled = true
+  config.lograge.custom_payload do |controller|
+    request = controller.request
+    {
+      current_user_id: controller.try(:current_user).try(:id),
+      host: request.host,
+      path: request.path,
+      query_string: request.query_string,
+      referer: request.referer,
+      user_agent: request.user_agent
```

Comment:
do we want this in our logs? or what information do we want?

Reply:
Is possible to add the time info?

Reply:
good question. In the lograge docs it says
>It doesn't include any timestamp by default, instead it assumes you use a proper log formatter instead.

And this is what I see for example in AWS Cloudwatch:
![image](https://github.com/rootstrap/rails_api_base/assets/6373536/9ed8478f-2167-45d0-8ac0-307012aca22b)

Reply:
Few questions here:
- Can que add the IP address? Might be useful to detect bots attacks or similar
- Would query_string include all the request params? I mean, from url and from a form/post request body?

Reply:
- IIRC IP addresses are considered to be PII which goes against GDPR. I'd like this base app to be GDPR-compliant by default
- query_string only have the query params in the URL